modules/log/birth/farm_birth.post_update.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/**
+ * @file
+ * Post update hooks for the farm_birth module.
+ */
+
+/**
+ * Override the birth log asset label and description.
+ */
+function farm_birth_post_update_override_birth_asset_label_description(&$sandbox) {
+
+  // Override the asset field label and description on birth logs to make it
+  // clear that it should be used to reference the child assets that were born.
+  $config = \Drupal::service('entity_field.manager')->getBaseFieldDefinitions('log')['asset']->getConfig('birth');
+  $config->set('label', 'Children');
+  $config->set('description', 'Which child assets were born?');
+  $config->save();
```

Comment:
Oh hmm I would have guessed that we create a new `base_field_override` config entity. This must be happening behind the scenes. But is it possible to do it more directly?

Reply:
Ah good question. That may be possible too. :thinking: 

My code was based on this: https://www.previousnext.com.au/blog/overriding-base-field-labels-and-descriptions-of-drupal-entities-without-custom-code

I'm not sure what the "right" / "best" way to do it is. Perhaps there's an argument that going through `entity_field.manager` has some benefits because it is doing it from a higher level? It probably doesn't make a big difference either way. The nice thing about this is that we only need to set `label` and `description`. Constructing a `BaseFieldOverride` entity would require defining *all* the properties of that entity, which we get for free via `getConfig('birth')`.